.github/ISSUE_TEMPLATE/new-document.md

```diff
@@ -0,0 +1,14 @@
+---
+name: New document
+about: Request a new document for the Clear Linux* project
+
+---
+
+**Do you think Clear Linux documentation needs a new document? Please describe.**
+Please provide a clear and concise description of the title and content. Identify the target audience: Developer; System Administrator; or Basic User.
```

Comment:
Did we drop our requirement for wordwrap at 78 characters?

Also ask if the requested doc would be a guide, a reference or a tutorial.

Reply:
Line length found here: https://clearlinux.org/documentation/clear-linux/reference/collaboration/documentation/basic

See latest push for identifying doc type.

Reply:
@tpleavitt. Please advise if this meets your criteria.

Reply:
Lines 10 and 11 of the doc get it!  Thanks Michael